images/ose-cli-artifacts.yml

```diff
@@ -30,4 +30,6 @@ owners:
 - aos-master@redhat.com
 konflux:
   vm_override:
-    aarch64: linux-d160/arm64
+    x86_64: linux-d160-m2xlarge/amd64
```

Comment:
`linux-d160-c2xlarge/amd64` or a higher VM flavor might be more apt here. I can raise a PR to create one.